calamari_ocr/scripts/predict.py

```diff
@@ -41,13 +41,16 @@ def run(args):
 
     # load files
     input_image_files = glob_all(args.files)
+    
```

Comment:
remove the lines

Reply:
done

Reply:
There is still a line (`git diff calamari_ocr/scripts/predict.py` should show no changes)